cli/tools/test.rs

```diff
@@ -508,21 +502,101 @@ impl TestReporter for PrettyTestReporter {
         format!(" ({} steps)", count)
       }
     };
+
+    println!();
+    println!("{} {}", colors::gray("test result:"), status);
     println!(
-      "\ntest result: {}. {} passed{}; {} failed{}; {} ignored{}; {} measured; {} filtered out {}\n",
-      status,
+      "{}      {}{}",
+      colors::gray("passed:"),
       summary.passed,
-      get_steps_text(summary.passed_steps),
-      summary.failed,
-      get_steps_text(summary.failed_steps + summary.pending_steps),
-      summary.ignored,
-      get_steps_text(summary.ignored_steps),
-      summary.measured,
-      summary.filtered_out,
-      colors::gray(
-        format!("({})", display::human_elapsed(elapsed.as_millis()))),
+      get_steps_text(summary.passed_steps)
     );
+    if summary.failed > 0 {
+      println!(
+        "{}      {}{}",
+        colors::gray("failed:"),
+        summary.failed,
+        get_steps_text(summary.failed_steps)
+      );
+    }
+    if summary.ignored > 0 {
+      println!(
+        "{}     {}{}",
+        colors::gray("ignored:"),
+        summary.ignored,
+        get_steps_text(summary.ignored_steps)
+      );
+    }
+    if summary.filtered_out > 0 {
+      println!("{}    {}", colors::gray("filtered:"), summary.filtered_out);
+    }
```

Comment:
Keep in mind, that "failed", "ignored" and "filtered out" counts are conditional and are displayed only if the count of that category is greater than zero. Not sure if this is desirable or not. Options welcome

Reply:
Probably a good thing. There's very little reason to keep the cluttering messages around when they convey exactly no information.